### DISCUSSION TITLE
Pool "public-pool.airdns.org" not stable at the moment?

### DISCUSSION THREAD

**Discussion by onlinecrash24** [BitMaker-hub/NerdMiner_v2#111] [General]
The pool "public-pool.airdns.org" does not seem to run stable and the web interface is not consistently accessible. Do you also have this problem?

**Comment by BitMaker-hub** [comment]
Unfortunately the pool is experiencing some investability now but pool admin is trying to fix it.
We suggest to move to other pools until stabilized

**Comment by 570RM7R4C3R** [comment]
I've created a solo pool here : https://vkbit.com based on public-pool code.
Everyone is welcome, hope it helps.

**Reply by Gisela3112** [reply]
Hello,
I'm new here and I have some questions: how can I create an account at vkbit.com? I have already send a request to the support, but without any answer. And how can I connect to your solo pool?
Thank you
Gisela

**Reply by 570RM7R4C3R** [reply]
Hello,
You don't need an account to start mining or test the vkbit.com pool.
The account is only used for the NOSTR widgets.
Thank you.
